### PR TITLE
Turn off VMODL install

### LINKED ISSUE #943: Need kill switch for vmodl support
We need to have kill switch for VMODL support, and for *not* recycling vsanmgmgt svc on our svc start. This will prevent unneeded impact on VSAN unless VMODL need to be turned on. 

I suggest doing it via installing the vmodl files somewhere else  and modifying vmdk-opsd -to  NOT recycle vsanmgmt svc by default, and adding  `vmdk-opsd vmodlon` and `vmodloff` manipulate the vmodl py files location and vsanmgmtd restart

Putting in 1.12 for discussion 

//CC @pdhamdhere @shaominchen

### PR DESCRIPTION
**Pull request by msterin**
Fixes #943 
This PR removes VMODL support from .vib and comments out vsanmgmtd restart on daemon start. 
SInce we do not use (nor publish) vmodl yet, we need to minimize impact on other components. 

//CC @pdhamdhere @shaominchen

### CODE REVIEW

**Comment by shuklanirdesh82** [comment]
@msterin 

can you please help to check for the following error cause?

```
Running unit tests in /tmp/vmdk_ops_unittest11530/vmodl_test.py...
Traceback (most recent call last):
  File "/tmp/vmdk_ops_unittest11530/vmodl_test.py", line 51, in <module>
    import VsphereContainerService
ImportError: No module named 'VsphereContainerService'
```

**Comment by shaominchen** [comment]
We need to disable vmodl_test as well since VMODL is turned off.

**Comment by msterin** [comment]
@shuklanirdesh82 @shaominchen  - thanks for the catch. Fixed.

File: esx_service/vmodl/Makefile [hunk 1]
```diff
@@ -70,11 +72,17 @@ test: deploy
...
-ifdef INSTALL_VMODL
 	echo GENERATING VMODL PAYLOAD...
 	mkdir -p ../$(VMDKOPS_PAYLOAD)$(TARGET2) ../$(VMDKOPS_PAYLOAD)$(TARGET3)
 	cp $(PYFILES) $(TESTFILES) ../$(VMDKOPS_PAYLOAD)$(TARGET2)
 	cp $(PYFILES) $(TESTFILES) ../$(VMDKOPS_PAYLOAD)$(TARGET3)
+
 else
-	@echo Skipping VMODL payloads. See https://github.com/vmware/docker-volume-vsphere/pull/975 for details.
+
+deploy test copy_to_payload:
```
Comment (shaominchen) [inline esx_service/vmodl/Makefile]: Why do we need this line?

Comment (msterin) [inline esx_service/vmodl/Makefile]: to make sure misc. CI scripts invoking 'make test' or 'make deploy' or 'make copy_to_payload' complete successfully with the proper message

Comment (shaominchen) [inline esx_service/vmodl/Makefile]: OK, make sense.

**Pull request merged**